.vscode/settings.json

```diff
@@ -8,6 +8,11 @@
   "python.testing.unittestEnabled": false,
   "python.testing.pytestEnabled": true,
   "[python]": {
+    "editor.formatOnSave": true,
```

Comment:
Added auto-format on save. I find it useful because my editor no longer constantly compains at me, but I can remove if others find it annoying.